core/blockchain/sequencelock.go

```diff
@@ -42,7 +42,7 @@ func (b *BlockChain) calcSequenceLock(node *blockNode, tx *types.Tx, view *UtxoV
 	msgTx := tx.Transaction()
 	//TODO, revisit the tx version for lock time
 	enforce := isActive && msgTx.Version >= 2
-	if !enforce || msgTx.IsCoinBase() {
+	if !enforce || msgTx.IsCoinBase() || tx.IsDuplicate {
```

Comment:
the seqlock design need to review, we cannot take the assumption that if the tx is duplicated then the seqlock will not apply